queue_services/account-mailer/src/account_mailer/email_templates/payment_due_notification.html

```diff
@@ -1,12 +1,18 @@
-# Payment Due Today
+# Payment Due for {{ account_number }}: {{ account_name_with_branch }}
 
-## {{ account_number }}: {{ account_name_with_branch }}
+{% if short_name_links_count > 1 %}
+    There's an amount owing on your account for your {{ statement_month }} statement (#{{ statement_number }}), which is due today {{ due_date }}.
+{% else %}
+    There's an amount owing of {{ total_amount_owing }} on your account for your {{ statement_month }} statement (#{{ statement_number }}), which is due today {{ due_date }}.
```

Comment:
Isn't it possible to combine these a bit more?

Reply:
GPT:

```
{% set owing_message = "There's an amount owing" %}
{% set amount_details = '' if short_name_links_count > 1 else ' of {{ total_amount_owing }}' %}
{{ owing_message }}{{ amount_details }} on your account for your {{ statement_month }} statement (#{{ statement_number }}), which is due today {{ due_date }}.
```

Reply:
Possibly, I prefer the whole sentence rather trying to combine fragments though when we have to modify it.

Reply:
ok, then you'll be modifying it twice if the base sentence needs to change